.github/dependabot.yml

```diff
@@ -1,9 +1,16 @@
 version: 2
 updates:
 - package-ecosystem: cargo
+  versioning-strategy: lockfile-only
```

Comment:
Unfortunately, we can not have two separate sections one for grouped Cargo.lock updates and another for one-PR-per-update for breaking releases of direct upstream dependencies.